client/src/App.js

```diff
@@ -104,6 +105,7 @@ function App() {
             <Route element={<PrivateRoute roles={[roles.admin, roles.teacher]}/>}>
               <Route path='/fetch-instances/:courseId' element={<FetchInstancesView/>}/>
               <Route path='/edit-instance/:instanceId' element={<EditInstanceView/>}/>
+              <Route path='/select-formula' element={<SelectFormulaView/>}/> { /* Path will be replaced with '/select-formula/:instanceId/:assignmentId' once component is connected to a page */ }
```

Comment:
Maybe add the comment on its own line on the top so that the line isn't that long?